database/things/cokescript.pldb

```diff
@@ -0,0 +1,14 @@
+title CokeScript
+appeared 2015
+type pl
```

Comment:
should this be 'pl' or 'esolang'?

Reply:
Doesn't look like one to me.
https://github.com/batiste/CokeScript#cokescript